pkg/kubectl/cmd/apply.go

```diff
@@ -38,6 +38,7 @@ import (
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/apimachinery/pkg/util/strategicpatch"
 	"k8s.io/apimachinery/pkg/util/wait"
+	openapi "k8s.io/kube-openapi/pkg/util/proto"
```

Comment:
why is it called `proto`?

Reply:
These utilities works with proto version of openapi spec as far as I can tell. So they should be called that I guess.

Reply:
Fair enough :-)